scripts/migration/migrate_menas.py

```diff
@@ -0,0 +1,326 @@
+#!/usr/bin/python3
+
+import argparse
+import secrets  # migration hash generation
+from minydra.dict import MinyDict  # dictionary with dot access
+from pymongo import MongoClient
+from pymongo.database import Database
+from pymongo.write_concern import WriteConcern
+from pymongo.read_concern import ReadConcern
+
+
+# Default configuration
+# =====================
+
+defaults = MinyDict({
+    'verbose': False,
+    'dryrun': False,
+    'target_db_name': "menas_migrated"
+})
+
+migration_hash = secrets.token_hex(3)  # e.g. 34d4e10f
+
+SOURCE_DB_NAME = "menas"
+
+# Constants
+NOT_LOCKED_MONGO_FILTER = {"$or": [
+    {"locked": False},  # is not locked, or
+    {"locked": {"$exists": False}}  # or: there is no locking info at all
+]}
+EMPTY_MONGO_FILTER = {}
+
+
+def parse_args():
```

Comment:
Do we add return types?

Reply:
I added param types and return types in the method signatures. The only place that I am unsure of is how to type the function-type params.